### DISCUSSION TITLE
How to configure the xml report so, that only annoted tests are included

### DISCUSSION THREAD

**Discussion by straiker** [Xray-App/xray-testng-extensions#20] [Q&A]
Hi! 

I've moved my tests from JUnit to TestNG and am having a hard time to correctly configure the report. For JUnit there was a property that stated that it should include only annoted tests

report_only_annoted_tests = true

but this does not seem to be working for TestNG. Is there a way to actually make it work?

**Comment by bitcoder** [comment]
Hi @straiker , that configuration option doesn't exist in this project as you can see on the [README file](https://github.com/Xray-App/xray-testng-extensions/blob/master/README.md).
I've created a feature request #21 21, please leave feedback on it.

Btw, are you using the Xray JSON reporter (through XrayJsonReporter) or the standard TestNG XML output (through the XrayListener)?

**Comment by bitcoder** [comment]
@straiker , can you please try the 0.3.0-beta build, on the [report_only_annotated branch](https://github.com/Xray-App/xray-testng-extensions/tree/report_only_annotated)? This applies only for the XrayJsonReporter

**Comment by bitcoder** [comment]
Made this (for the Xray JSON reporter) as part of the 0.3.0 release: https://github.com/Xray-App/xray-testng-extensions/releases/tag/0.3.0
